src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -62,17 +58,17 @@ const styles = StyleSheet.create({
     marginBottom: SIZES.base
   },
   searchPressableContainer: {
-    flex: 1
+    flex: 1,
+    justifyContent: 'center'
   },
   wrapperSearch: {
-    flex: 1,
     flexDirection: 'row',
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
     marginLeft: 20,
     marginRight: 12,
-    borderRadius: 8,
+    borderRadius: 12,
     alignSelf: 'center',
-    height: 36
+    height: 28
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the search container has been modified. The `flex` property has been set to 1 and `justifyContent` has been set to 'center'. The background color has been changed and the border radius has been increased. The height has also been decreased. Ensure that these changes improve the UI and do not cause any layout issues.

```diff
- flex: 1,
- backgroundColor: COLORS.lightgrey,
- borderRadius: 8,
- height: 36
+ flex: 1,
+ justifyContent: 'center',
+ backgroundColor: COLORS.gray100,
+ borderRadius: 12,
+ height: 28
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->